lighthouse-core/gather/gather-runner.js

```diff
@@ -402,6 +401,7 @@ class GatherRunner {
           // If the main document redirects, we'll update this to keep track
           url: options.requestedUrl,
           settings: options.settings,
+          hostUserAgent: baseArtifacts.HostUserAgent,
```

Comment:
I was parsing the user agent over in #5893, maybe after this we should do a refactor for some useful `host.chromeVersion`, `host.isMobile`, `host.os`?